packages/klevu-ui/src/components/klevu-product-query-popup/klevu-product-query-popup.tsx

```diff
@@ -252,13 +258,17 @@ export class KlevuProductQueryPopup {
     const timeout = setTimeout(() => {
       this.showLoadingSorry = true
     }, 4000)
-    await this.session?.query(
-      {
-        message: message,
-        klevuSettings: this.settings,
-      },
-      "send"
-    )
+    await this.session
+      ?.query(
+        {
+          message: message,
+          klevuSettings: this.settings,
+        },
+        "send"
+      )
+      .catch((err) => {
+        this.sendMessageError = true
```

Comment:
we should reset to false when the user sends message again

Reply:
add reset to onMessage function